swagger/v1/prosecution_case.json

```diff
@@ -97,6 +97,9 @@
               },
               {
                 "$ref": "offence.json#/definitions/resource"
+              },
+              {
+                "$ref": "hearing_summary.json#/definitions/resource"
```

Comment:
I thought that this wasn't available in v1?

Reply:
It is @arthurashman

Reply:
It's hearings that aren't.

Reply:
👍